sandbox/src/mocks/stu3/changeShortlistAndSendForTriage/responses/MinimalRequest.json

```diff
@@ -200,6 +216,50 @@
                                 "status": "accepted"
                             }
                         ]
+                    },
+                    {
+                        "indexed": "2021-10-13T12:51:31.874Z",
+                        "meta": {
+                            "profile": [
+                                "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-DocumentReference-1"
+                            ]
+                        },
+                        "description": "Description for TXT File",
+                        "id": "DocumentReference-70000",
```

Comment:
In the example request, we have "code": "NEED_TO_ADD_LATER", so there shouldn't be an attachment.

If I try to call A032 on a UBRN with a referral letter I get:
"diagnostics": "'Shortlist/Intention To Add Referral Letter' is set to NEED_TO_ADD_LATER, but a referral letter has already been added to the Appointment Request."

Reply:
Maybe change it to "NOT_INTENDING_TO_ADD".

Reply:
Changed to "NOT_INTENDING_TO_ADD"